lib/mongodb.js

```diff
@@ -732,11 +732,31 @@ MongoDB.prototype.buildSort = function(model, order) {
   return sort;
 };
 
+var DEG2METER = 111194.7618858386112;
+var RAD2METER = 6370990.56;
+var FEET2METER = 0.3048;
+
+function convertToMeter(distance, unit) {
+  switch (unit) {
+    case 'meters':
+      return distance;
+    case 'kilometers':
+      return distance * 1000;
+    case 'miles':
+      return distance * 1600;
+    case 'feet':
+      return distance * FEET2METER;
+    case 'radians':
+      return distance * RAD2METER;
```

Comment:
I'm no GIS or math wiz so maybe I'm missing something but wouldn't the radians and degree conversions depend on the relative coordinate location as opposed to constant conversion value?